### ISSUE TITLE
Codeblock `svg` not rendering properly in dark/oled theme

### ISSUE THREAD

**Issue by hawkinslabdev** [melosso/teatime#3]
E.g. `<img src="/icons/docker.svg" class="tab-icon" alt="" aria-hidden="true" loading="lazy">` is rendered white in OLED/black.

**Comment by hawkinslabdev** [comment]
Resolved in 6bb812ea97044baf3958da6fdc267ccf563806df.
